monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,22 @@
+[id="accessing-monitoring-for-user-defined-projects"]
+= Accessing monitoring for user-defined projects
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: accessing-monitoring-for-user-defined-projects
+
+toc::[]
+
+By default, centralized monitoring for user-defined projects and platform monitoring are enabled. You can monitor your own projects in {product-title} without the need for an additional monitoring solution.
+
+The monitoring of user-defined projects cannot be disabled.
+
+The `dedicated-admin` user has default permissions to configure and access monitoring for user-defined projects.
+
+[NOTE]
+====
+Custom Prometheus instances and the Prometheus Operator installed through Operator Lifecycle Manager (OLM) can cause issues with user-defined workload monitoring if it is enabled. Custom Prometheus instances are not supported.
```

Comment:
I changed it. But I had checked with SRE and was told that both are correct because it's monitoring workloads for user-defined projects. But I changed it to projects here because that's splitting hairs too close to GA :)